test/csx.coffee

```diff
@@ -0,0 +1,616 @@
+# We usually do not check the actual JS output from the compiler, but since
+# CSX is not readily supported by Node, we do it in this case
+eqCSX = (cs, js) ->
```

Comment:
There’s already a `toJS` function in global scope. Can we please just use it, and follow the pattern of the modules tests?